tools/ci_build/github/linux/run_build.sh

```diff
@@ -62,14 +62,14 @@ else
     if [ $BUILD_DEVICE = "gpu" ]; then
         if [ $BUILD_OS = "manylinux2010" ]; then
             python3 $SCRIPT_DIR/../../build.py --build_dir /build \
-                --config Debug Release $COMMON_BUILD_ARGS \
+                --config Release $COMMON_BUILD_ARGS \
```

Comment:
Curious, is "RelWithDebInfo" better than "Release"?

Reply:
I don't know the exact difference between them. I think we'll need to read cmake's code to figure it out.